bin/compile

```diff
@@ -29,6 +32,11 @@ RUNTIME_FILE="runtime.txt"
 
 echo "-----> Export Python version from Poetry to Heroku $RUNTIME_FILE file"
 
+if [ "$DISABLE_POETRY_CREATE_RUNTIME_FILE" ] && [ "$DISABLE_POETRY_CREATE_RUNTIME_FILE" != "0" ]; then
+  echo "-----> Skipping generation of $RUNTIME_FILE file from poetry.lock"
```

Comment:
Not necessarily related to this PR, but it would be nice if you could add something like
```sh
function log() {
  echo "-----> $@"
}
```
and change all logging accordingly, but it's up to you whether you want to do it or not.

Reply:
I like this idea, but suggest we merge this PR and refactoring go into the next/another one.